### PR TITLE
KFSPTS-12304 change ezra service to use document service for routing …

### PR DESCRIPTION
**Pull request by db483**
…documents

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java [hunk 1]
```diff
@@ -456,7 +456,8 @@ private void routeAgencyDocument(Agency agency, Agency oldAgency) {
 		agencyDoc.setNewMaintainableObject(agencyMaintainable);
 		try {
 			documentService.saveDocument(agencyDoc);
-			agencyDoc.getDocumentHeader().getWorkflowDocument().route("Automatically created and routed");
+			GlobalVariables.getMessageMap().clearErrorMessages();
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java]: Further above, workflow exceptions are caught.  Should those exceptions prevent the document from routing?  If so, clearing the error messages may prevent that.

File: src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java [hunk 2]
```diff
@@ -490,8 +490,8 @@ private void routeAwardDocument(Award award, Award oldAward) {
 		awardDoc.getNewMaintainableObject().setBusinessObject(award);;
 		try {
 			documentService.saveDocument(awardDoc);
-			//documentService.routeDocument(awardDoc, "Automatically created and routed", null);
-			awardDoc.getDocumentHeader().getWorkflowDocument().route("Automatically created and routed");
+			GlobalVariables.getMessageMap().clearErrorMessages();
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java]: same concern as I mentioned in agency

File: src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java [hunk 3]
```diff
@@ -519,7 +519,8 @@ private void routeProposalDocument(Proposal proposal) {
 		proposalDoc.getNewMaintainableObject().setBusinessObject(proposal);
 		try {
 			documentService.saveDocument(proposalDoc);
-			proposalDoc.getDocumentHeader().getWorkflowDocument().route("Automatically created and routed");
```
Comment (jhulslander) [inline src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java]: same concern as I mentioned in agency

File: src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java (hunk 1 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java]: I think that the Workflow exceptions would still be thrown, they are not cleared with the clearErrorMessages

Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/module/ezra/service/impl/EzraServiceImpl.java]: oK, I shall merge

**Pull request merged**